From f7f7de2f083e976d51d888cdb0616650e68361a2 Mon Sep 17 00:00:00 2001
From: Bernhard Messerklinger <bernhard.messerklinger@br-automation.com>
Date: Mon, 3 Sep 2018 10:17:35 +0200
Subject: [PATCH 11/24] serial: mxc: Add match string for i.mx6 quad/dual lite
 serial

Signed-off-by: Bernhard Messerklinger <bernhard.messerklinger@br-automation.com>
Reviewed-by: Hannes Schmelzer <oe5hpm@oevsv.at>
Tested-by: Hannes Schmelzer <oe5hpm@oevsv.at>
(cherry picked from commit 4684fa8bdd7bb38ef54cc4c60ec12c649536f288)
---
 drivers/serial/serial_mxc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/serial/serial_mxc.c b/drivers/serial/serial_mxc.c
index c96a04ff57..8fa63b715f 100644
--- a/drivers/serial/serial_mxc.c
+++ b/drivers/serial/serial_mxc.c
@@ -368,6 +368,7 @@ static int mxc_serial_ofdata_to_platdata(struct udevice *dev)
 static const struct udevice_id mxc_serial_ids[] = {
 	{ .compatible = "fsl,imx6ul-uart" },
 	{ .compatible = "fsl,imx7d-uart" },
+	{ .compatible = "fsl,imx6q-uart" },
 	{ }
 };
 #endif
-- 
2.17.1

